kops

```diff
@@ -255,14 +263,17 @@ def _wait_for_cluster(kubecfg_path):
         else:
             break
 
-
-def _cmd_create(args):
-    """Invoke kops to create cluster."""
-    cluster_name = '{}.{}'.format(args.category, args.name)
+def _check_existing_cluster(args, cluster_name):
+    """Find out if a cluster with the same name exists and if so, asign a number to make it different"""
     re_cluster = re.compile(r'{}-(\d+)'.format(cluster_name), re.I)
     existing_clusters = _run_kops(
-        ['get', 'clusters'], args, ignore_dry_run=True, get_output=True
-    ).splitlines()
+        ['get', 'clusters'], args, ignore_dry_run=True, get_output=True, tolerate_error=True
+    )
+
+    if existing_clusters is None:
+        return 0
+
+    existing_clusters.splitlines()
```

Comment:
I don't think this will work, since `str.splitlines` doesn't mutate, but returns a new string.